src/App.test.jsx

```diff
@@ -10,7 +10,7 @@ describe('App', () => {
     screen.getByText(/loading/i);
 
     // loads film --- 10 films per page
-    const film = await screen.findByAltText('film');
+    const film = await screen.findByText(/list/i);
```

Comment:
This test could be much more comprehensive.  You are verifying that the heading "List" is on the page, but isn't there also a whole list that loads in after the loading state is complete?  You want your tests to account for everything on the page if you are testing the entire App.  If you were testing just a header component, this test would be more appropriate.